docs/en/stack/security/authorization/built-in-roles.asciidoc

```diff
@@ -111,6 +111,14 @@ user has access to only their own reports. Reporting users should also be
 assigned the `kibana_user` role and a role that grants them access to the data 
 that will be used to generate reports.
 
+[[built-in-roles-snapshot-user]] `snapshot_user`::
+Grants the necessary privileges to create snapshots of **all** the indices and
+to view their metadata. This role enables users to view the configuration of
+existing snapshot repositories and of the snapshot details. It does not enable
+users to change or restore any of them. It also does not enable users to change
```

Comment:
The "them" is a little unclear here. Is it just "snapshots", "snapshot repositories", or "snapshot details"?

Reply:
All of them. This is poorly worded. I tried to say a lot in a few words and it did not turn up well.
I have pushed https://github.com/elastic/stack-docs/pull/212/commits/6ba2c46d80551ee5ad8159fdddea8006dee748ac